contracts/root/predicates/ERC721Predicate.sol

```diff
@@ -23,17 +23,17 @@ contract ERC721Predicate is IPredicate {
 
   constructor(address _withdrawManager) public IPredicate(_withdrawManager) {}
 
-  function startExit(bytes memory data)
+  function startExit(bytes memory data, bytes memory exitTx)
```

Comment:
Create one predicate interface and please follow it. You can use `address predicate, bytes memory data` or equivalent

Reply:
We don't need `address predicate`, because we are invoking `startExit` directly in the predicate contract. Going with `function startExit(bytes memory data, bytes memory exitTx)` where `data` is predicate context dependant and `exitTx` is the in-flight tx. Any other suggestion?